docs/anomaly-detection/docs/README.md

```diff
@@ -0,0 +1,27 @@
+---
+sort: 1
+title: Docs
```

Comment:
rename to something more meaningful, because the documentation site contains documentation

Reply:
renamed to `components` as that's what VM anomaly detection consists of, as well as it defines correspondent parts of service config